deploy/smcp-templates/servicemesh

```diff
@@ -0,0 +1,40 @@
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+spec:
+  template: base
+  istio:
+    global:
+      hub: registry.redhat.io/openshift-service-mesh
+      proxy:
+        image: proxyv2-rhel8
+      proxy_init:
+        image: proxy-init-rhel8
+    grafana:
+      image: grafana-rhel8
+    galley:
+      image: galley-rhel8
+    tracing:
+      jaeger:
+        hub: registry.redhat.io/distributed-tracing-tech-preview
+      elasticsearch:
+        hub: registry.centos.org/rhsyseng
+    mixer:
+      image: mixer-rhel8
+    pilot:
+      image: pilot-rhel8
+    security:
+      image: citadel-rhel8
+    istio-cni:
+      image: istio-cni-rhel8
+    kiali:
+      tag: 1.0.0
+    sidecarInjectorWebhook:
+      image: sidecar-injector-rhel8
+    gateways:
+      istio-ingressgateway:
+        ior_image: istio-ior-rhel8
+    prometheus:
+      image: prometheus-rhel8
+  threeScale:
+    hub: registry.redhat.io/openshift-istio-tech-preview
```

Comment:
@brian-avery remove `openshift-istio-tech-preview` and replace with `openshift-service-mesh` here and elsewhere?